### PR TITLE
Support cap-add and cap-drop

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

File: src/test/java/com/spotify/docker/client/messages/HostConfigTest.java [hunk 1]
```diff
@@ -65,6 +66,14 @@ public void testJsonOnFailure() throws Exception {
     assertThat(hostConfig.restartPolicy(), is(HostConfig.RestartPolicy.onFailure(5)));
   }
 
+  @Test
+  public void testCapAddAndDrop() throws Exception {
```
Comment (davidxia) [inline src/test/java/com/spotify/docker/client/messages/HostConfigTest.java]: A bit of a dumb test, but we can't easily mock WebTarget in DefaultDockerClient right now...

**Comment by mattnworb** [comment]
@davidxia I think a good strategy for testing it would be to use the `DefaultDockerClient` constructor that takes a `RSClientBuilderWrapper` and have that return a mock jersey client.

**Comment by mattnworb** [comment]
I think #376 would help with this

**Comment by davidxia** [comment]
@mattnworb Is this PR dependent on #376? Seems like it's not, but #376 seems good to have nonetheless.

**Comment by mattnworb** [comment]
is this no longer WIP? LGTM as is.

**Comment by davidxia** [comment]
Yup, it's done. I just updated the commit message.

**Pull request merged**